wicket-examples/src/main/java/org/apache/wicket/examples/websocket/JSR356Application.java

```diff
@@ -59,11 +66,22 @@ public void init()
 
 		getSharedResources().add(ChartWebSocketResource.NAME, new ChartWebSocketResource());
 
+		final WebSocketSettings webSocketSettings = WebSocketSettings.Holder.get(this);
+
+		webSocketSettings.setSocketSessionConfigurer(new IWebSocketSessionConfigurer() {
```

Comment:
you can be cool and use lambda here :-)

Reply:
I was not sure what was the accepted style in examples... we use extensively lambdas in our project. I will do.